r/R/dplyr-funcs-type.R

```diff
@@ -292,3 +293,18 @@ register_bindings_type_elementwise <- function() {
     is_inf & !call_binding("is.na", is_inf)
   })
 }
+
+register_bindings_type_format <- function() {
+  register_binding("format", function(x, ...) {
+    if (!inherits(x, "Expression")) {
+      return(format(x, ...))
+    }
```

Comment:
How does this sound? Since this is a bit unusual that we _aren't_ quickly converting as scalar to Arrow and then executing, having a comment as to why is good to have.

Reply:
Sure. Sounds good to me.